src/ui/src/js/controllers/command_view.js

```diff
@@ -155,7 +155,29 @@ export default function commandViewController(
       };
     }
 
-    RequestService.createRequest(newRequest).then(
+    let isFormData = false;
+    const fd = new FormData();
+    for (const i in $scope.command.parameters) {
+      if (
```

Comment:
This read as slightly confusing to me, since if the first if statement is true, the second one is guaranteed to also be true.  Could this just be:

```javascript
if ($scope.command.parameters[i].type.toLowerCase() === "bytes") {
    if (!isFormData) {
      fd.append("request", JSON.stringify(newRequest));
      isFormData = true;
    }
    fd.append(
      $scope.command.parameters[i].key,
      document.getElementById($scope.command.parameters[i].key).files[0]
    );
}
```